src/components/EnvoyFleets/EnvoyFleetInfo/APIs/APIs.tsx

```diff
@@ -0,0 +1,44 @@
+import {useEffect, useState} from 'react';
+
+import {useAppSelector} from '@redux/hooks';
+
+import * as S from './styled';
+
+interface ApisTableDataSourceItem {
+  key: string;
+  name: string;
+  namespace: string;
+}
+
+const columns = [
+  {title: 'Name', dataIndex: 'name', key: 'name', width: '50%'},
+  {title: 'Namespace', dataIndex: 'namespace', key: 'namespace', width: '50%'},
+];
+
+const APIs: React.FC = () => {
+  const apis = useAppSelector(state => state.main.selectedEnvoyFleet?.apis);
+
+  const [dataSource, setDataSource] = useState<ApisTableDataSourceItem[]>([]);
+
+  useEffect(() => {
```

Comment:
does this have to be in a useEffect and use some state? it looks to me that it can simply be a useMemo